src/helper/main.js

```diff
@@ -139,6 +142,11 @@ import { logAndDie, helpMessage, databases } from './helperUtil';
                 for (const [id, response] of newAutoResponces) {
                     if (id !== 'holder') responseDb.set(id, response);
                 }
+
+                console.log('Storing key data...');
+                for (const key of newKeys) {
+                    if (key.value !== 'holder') keyDb.set(key.value, key);
```

Comment:
Wait... so the keydb is structured as `'key' => { value: 'key' }` ???
I know that is probably the best solution but damn... document based dabases aint the best for this :D
I'll approve it anyway.